### PR TITLE
Update membership to clarify who can merge approved PRs.

### LINKED ISSUE #71: Update membership to clarify who can merge approved PRs.
https://github.com/open-telemetry/community/blob/master/community-membership.md

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
Fixes #71

### CODE REVIEW

File: community-membership.md [hunk 1]
```diff
@@ -181,6 +181,7 @@ The following apply to the subproject for which one would be an owner.
 - Make and approve technical design decisions for the subproject.
 - Set technical direction and priorities for the subproject.
 - Define milestones and releases.
+  - Decides on when PRs are merged to control the release scope.
```
Comment (yurishkuro) [inline community-membership.md]: is there indentation issue?

Comment (SergeyKanzhelev) [reply community-membership.md]: I made it a sub-item of a release control.

**Comment by bogdandrutu** [comment]
@lizthegrey this decision was made when the charters were defined. We can discuss that decision during governance meeting, this PR just clarifies a previous decision.

**Comment by lizthegrey** [comment]
+1 that "decide when things are merged" sufficiently allows delegation of the trivial merge approval while retaining control/authority.

**Pull request merged**